seed/data_importer/tasks.py

```diff
@@ -1552,17 +1550,21 @@ def pair_new_states(merged_property_views, merged_taxlot_views):
     # Not sure what the below cycle code does.
     cycle = chain(merged_property_views, merged_taxlot_views).next().cycle
 
-    tax_cmp_fmt = [('jurisdiction_tax_lot_id', 'custom_id_1'),
-                   ('custom_id_1',),
-                   ('normalized_address',),
-                   ('custom_id_1',),
-                   ('custom_id_1',)]
-
-    prop_cmp_fmt = [('lot_number', 'custom_id_1'),
-                    ('custom_id_1',),
-                    ('normalized_address',),
-                    ('pm_property_id',),
-                    ('jurisdiction_property_id',)]
+    tax_cmp_fmt = [
+        ('jurisdiction_tax_lot_id', 'custom_id_1'),
+        ('custom_id_1',),
+        ('normalized_address',),
+        ('custom_id_1',),
+    ]
+
+    prop_cmp_fmt = [
+        ('lot_number', 'custom_id_1'),
+        ('ubid',),
```

Comment:
@RDmitchell  --- this was really the only line that needed to change to support the ubid in pairing.